run/db.go

```diff
@@ -67,14 +62,8 @@ func prepareQuery(queryPath string, consulAddress string, template bool, variabl
 	return script, nil
 }
 
-// Reads the file ready for executing
-func readScript(file string) (string, error) {
-
-	scriptBytes, err := ioutil.ReadFile(file)
-	if err != nil {
-		return "", err
-	}
-	return string(scriptBytes), nil
+func readScript(f string) (string, error) {
+	return "", nil
```

Comment:
Can this function be removed now?  Does not appear to be doing anything anymore...